--- /dev/null	2017-06-20 22:44:58.033988373 +0000
+++ gitolite-3.6.7/Makefile	2017-07-08 14:26:22.684009258 +0000
@@ -0,0 +1,21 @@
+# borrowed from Fedora %install section
+
+all:
+
+
+install: 
+	# Directory structure
+	install -d $(DESTDIR)/var/lib/gitolite
+	install -d $(DESTDIR)/var/lib/gitolite/.ssh
+	install -d $(DESTDIR)/usr/bin
+	install -d $(DESTDIR)/usr/lib/perl5/5.24.0
+	install -d $(DESTDIR)/usr/share/gitolite
+	# Code
+	cp -pr src/lib/Gitolite $(DESTDIR)/usr/lib/perl5/5.24.0
+	echo "3.67" >src/VERSION
+	cp -a src/* $(DESTDIR)/usr/share/gitolite
+	ln -s /usr/share/gitolite/gitolite $(DESTDIR)/usr/bin/gitolite
+	# empty authorized_keys file
+	touch $(DESTDIR)/var/lib/gitolite/.ssh/authorized_keys
+
+.PHONY: install
+